asf_levies_model/consumers.py

```diff
@@ -0,0 +1,460 @@
+import pandas as pd
+
+
+class Consumer:
+    """
+    Represents a single energy consumer that pays gas and/or electricity bills.
+
+    Attributes:
+        name: The name of the energy consumer, e.g., Ofgem consumer archetype name.
+        net_annual_income: The average net annual income of represented consumer, in £.
+        net_income_decile: The income decile that the consumer's net annual income falls in.
+        main_heating_fuel: The name of the consumer's main heating fuel.
+        gas_consumption: Annual gas consumption, in MWh.
+        electricity_consumption: Annual electricity consumption, in MWh.
+        gas_tariff: Tariff object to be used to calculate the consumer's gas bill.
+        electricity_tariff: Tariff object to be used to calculate the consumer's electricity bill.
+        unmetered_fuel_spend: Annual amount spent on unmetered fuel (i.e., fuel that is not gas or electricity), in £.
+        scheme_eligible: Eligibilty for social support scheme.
+
+    Properties:
+        unit_cost_ratio: Electricity to gas unit cost ratio, considering any unit discounts from social support.
+        fuel_poverty_gap: Difference between total fuel spend and threshold of 10% of net annual income, in £.
+        electricity_subtotal_bill: Electricity subtotal bill, including VAT, before any social support adjustment, in £.
+        gas_subtotal_bill: Gas subtotal bill, including VAT, before any social support adjustment, in £.
+        electricity_bill: Electricity final bill, including VAT, after any social support adjustment, in £.
+        gas_bill: Gas final bill, including VAT, after any social support adjustment, in £.
+        dual_fuel_bill: Combined electricity and gas final bill, including VAT, after any social support adjustment, in £.
```

Comment:
We shouldn't call this a "dual fuel" bill as that's a specific type of tariff - this is just the combined gas and electricity bill.

Rename to reflect this.